gem/settings/base.py

```diff
@@ -229,7 +199,7 @@ def get_default_template(site_layout_base, site_layout):
 # We have multiple layouts: use `base`, `malawi` , `springster`, `rwanda`
 # 'chhaajaa' or 'yegna'
 # Change SITE_LAYOUT_BASE to switch between them.
-SITE_LAYOUT_BASE = environ.get('SITE_LAYOUT_BASE', 'base')
+SITE_LAYOUT_BASE = environ.get('SITE_LAYOUT_BASE', 'springster')
```

Comment:
Have you decided to default to the springster templates or is a change you made for testing that needs to be reversed?